internal/pkg/cli/init.go

```diff
@@ -109,6 +110,7 @@ func newInitOpts(vars initVars) (*initOpts, error) {
 	spin := termprogress.NewSpinner(log.DiagnosticWriter)
 	id := identity.New(defaultSess)
 	deployer := cloudformation.New(defaultSess, cloudformation.WithProgressTracker(os.Stderr))
+	iam := iam.New(defaultSess)
```

Comment:
Just noticed the block from L114-116 😂 Probably an orphan as a result of moving codes around. Can we remove that block plzz?